### PR TITLE
Default latest

### PR DESCRIPTION
**Pull request by SashaWeinstein**
# Pull latest version from data library
This is most substantial change we've made for this build as it effects all datasets. 
This change was prompted as I forgot to change the dataset version in datasets.yml during the dataloading step. Our other data products default to take the latest version of data from data library and can be adapted to take a particular version if need be. This seems to be a better approach so facdb was changed to default to latest version if no version is specified in datasets.yml

A change to a .sql file that should only be merged via another PR snuck in here, shouldn't mess anything up

### CODE REVIEW

File: facdb/datasets.yml [hunk 1]
```diff
@@ -34,32 +30,26 @@ datasets:
...
     scripts:
       - dsny_specialwastedrop.sql
 
   - name: dsny_textiledrop
-    version: 20210415
     scripts:
       - dsny_textiledrop.sql
 
   - name: dsny_leafdrop
-    version: 20210402
```
Comment (mbh329) [inline facdb/datasets.yml]: This won't work, it needs to be pointed to the older version

**Review by mbh329 — CHANGES_REQUESTED** [review]
Only thing is to change the dsny_leafdrop to point to the old version as I haven't fixed it yet

**Comment by td928** [comment]
after Max's comment about leafdrop is addressed should be good to go?

**Review by td928 — APPROVED** [review]

**Comment by td928** [comment]
After approval realize there is merge conflicts for some empty lines in .devcontainer/docker-compose.yml

**Comment by SashaWeinstein** [comment]
addressing merge conflicts now. removed 11 from postgres client in dockerfile, dealing with some changes to poetry lock. seeing if everything works now

**Review by mbh329 — APPROVED** [review]

**Comment by AmandaDoyle** [comment]
I apologize for not weighing in on this sooner.  Let's see how this works out.  The reason why a specific version was set for each dataset was because of the lengthy QA and iteration process, so if a dataset was updated during a review process it could cause us to have to update the ingestion process and/or introduce new anomalies in a QAQC report, further slowing the review process.

**Pull request merged**